### PR TITLE
[943] | Neeraj/Aditi | Introduce API to get Id and secret from gateway

### PR DESCRIPTION
**Pull request by imaditiagg**

### CODE REVIEW

File: src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java [hunk 1]
```diff
@@ -107,6 +110,27 @@ private BridgeServiceRequest bridgeServiceRequest(String id, String name, Servic
                 .then();
     }
 
+    public Mono<SupportBridgeResponse> getClientIdAndSecret(SupportBridgeRequest request, String token) {
```
Comment (mddubey) [inline src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java]: Aren't we generating a new one here? If yes then let's rename this to generateClientIdAndSecred

File: src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -33,4 +33,16 @@
...
     </changeSet>
+    <changeSet id="202009140505" author="Neeraj/Aditi">
+        <addColumn tableName="support_requests">
+            <column name="support_request_id" type="VARCHAR(50)">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="date_created" type="DATETIME" defaultValueDate="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_modified" type="DATETIME">
```
Comment (mddubey) [inline src/main/resources/liquibase.xml]: Shouldn't we specify `defaultValueDate ` here as well?

File: src/main/resources/liquibase.xml [hunk 3]
```diff
@@ -33,4 +33,16 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet id="202009140505" author="Neeraj/Aditi">
```
Comment (mddubey) [inline src/main/resources/liquibase.xml]: I think given we have not deployed this changelog anywhere yet, we could change the existing changelog of createTable.

File: src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java (hunk 1 above)
Comment (imaditiagg) [reply src/main/java/in/projecteka/devservice/clients/ServiceAuthenticationClient.java]: Done

File: src/main/resources/liquibase.xml (hunk 2 above)
Comment (imaditiagg) [reply src/main/resources/liquibase.xml]: Done

File: src/main/resources/liquibase.xml (hunk 3 above)
Comment (imaditiagg) [reply src/main/resources/liquibase.xml]: Done

**Review by mddubey — APPROVED** [review]

**Pull request merged**